src/lib.rs

```diff
@@ -485,6 +485,27 @@ impl OctocrabBuilder<NoSvc, DefaultOctocrabBuilderConfig, NoAuth, NotLayerReady>
         self
     }
 
+    /// Set the connect timeout.
+    #[cfg(feature = "timeout")]
```

Comment:
Does this really need its own feature flag? I think it's probably better to just have it included by default.